src/main/groovy/io/saagie/plugin/dataops/models/VariableMapper.groovy

```diff
@@ -0,0 +1,11 @@
+package io.saagie.plugin.dataops.models
+
+class VariableMapper {
```

Comment:
I don't understand the usage of this class, can you explain please?

Reply:
We use it to map the data retrieved inside the json that's contained in the zip file that we want to import, VariableMapper is similar to the input requested by the graphQL API that we use to create / update environment variable